abb_irb120_support/urdf/irb120_macro.xacro

```diff
@@ -0,0 +1,195 @@
+			</geometry>
+            <material name="yellow"/>
+		</collision>
+	</link>
+	<link name="${prefix}link_2">
+		<visual>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/visual/link_2.STL"/>
+			</geometry>
+            <material name="abb_orange"/>
+		</visual>
+		<collision>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/collision/link_2.STL"/>
+			</geometry>
+            <material name="yellow"/>
+		</collision>
+	</link>
+	<link name="${prefix}link_3">
+		<visual>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/visual/link_3.STL"/>
+			</geometry>
+            <material name="abb_orange"/>
+		</visual>
+		<collision>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/collision/link_3.STL"/>
+			</geometry>
+            <material name="yellow"/>
+		</collision>
+	</link>
+	<link name="${prefix}link_4">
+		<visual>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/visual/link_4.STL"/>
+			</geometry>
+            <material name="abb_orange"/>
+		</visual>
+		<collision>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/collision/link_4.STL"/>
+			</geometry>
+            <material name="yellow"/>
+		</collision>
+	</link>
+	<link name="${prefix}link_5">
+		<visual>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/visual/link_5.STL"/>
+			</geometry>
+            <material name="abb_orange"/>
+		</visual>
+		<collision>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/collision/link_5.STL"/>
+			</geometry>
+            <material name="yellow"/>
+		</collision>
+	</link>
+	<link name="${prefix}link_6">
+		<visual>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/visual/link_6.STL"/>
+			</geometry>
+			<material name="black">
+			    <color rgba="0 0 0 1"/>
+			</material>	
+		</visual>
+		<collision>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/collision/link_6.STL"/>
+			</geometry>
+            <material name="yellow"/>
+		</collision>
+	</link>
+	<link name="${prefix}tool0"/>
+	<!-- end of link list -->
+
+	<!-- joint list -->
+	<joint name="${prefix}joint_1" type="revolute">
+		<origin rpy="0 0 0" xyz="0 0 0"/>
+		<parent link="${prefix}base_link"/>
+		<child link="${prefix}link_1"/>
+		<limit effort="0" lower="-2.87979" upper="2.87979" velocity="4.36332"/>
+		<axis xyz="0 0 1"/>
+        <calibration rising="0" falling="0"/>
+		<dynamics damping="0" friction="0"/>
```

Comment:
Not sure whether we want to keep these elements in a basic urdf. Do you have any specific use for them?

@shaun-edwards ?